core/dbt/task/run.py

```diff
@@ -108,6 +108,22 @@ def track_model_run(index, num_nodes, run_model_result):
     if tracking.active_user is None:
         raise DbtInternalError("cannot track model run with no active user")
     invocation_id = get_invocation_id()
+    node = run_model_result.node
+    has_group = (
+        True if (node.resource_type == NodeType.Model and node.group is not None) else False
+    )
+    access = (
+        node.access.value
+        if (node.resource_type == NodeType.Model and node.access is not None)
+        else None
+    )
+    contract_enforced = (
+        node.contract.enforced
+        if (node.resource_type == NodeType.Model and node.contract)
+        else False
+    )
+    versioned = True if hasattr(node, "version") and node.version else False
```

Comment:
will we able to check if `node.resource_type == NodeType.Model` here once the versions changes are in?

Reply:
This is based off the versions branch, but "group" can actually be on other node types too, so I'll try hasattr on that one. Contract and access are only on models, right?

Reply:
> Contract and access are only on models, right?
right! version is also only on models